snap/snapcraft.yaml

```diff
@@ -45,6 +45,13 @@ package-repositories:
     key-id: 4D1BB29D63D98E422B2113B19334A25F8507EFA5
     url: http://repo.percona.com/pbm/apt
 
+slots:
+  logs:
+    interface: content
+    source:
+      read:
+        - $SNAP_COMMON/var/log/mongod
```

Comment:
@delgod should we read from other compenents? i.e. `mongos` or `pbm` or should we save that for later?